yugabyte/src/yugabyte/ysql/client.clj

```diff
@@ -33,42 +34,65 @@
    :connectTimeout (/ default-timeout 1000)
    :socketTimeout  (/ default-timeout 1000)})
 
+(defn- append-op-index
+  "Append /* <:op-index> */ to a given SQL statement or its part,
+  useful while digging through server-side logs.
+  If the given SQL contains a sequence, op-index will be appended to a first element."
```

Comment:
Better to make it clear: "If the given SQL contains a sequence, *commented* op-index will be appended to a first element."